src/types/publication_types/reportRegistration.types.ts

```diff
@@ -34,14 +32,6 @@ interface ReportReference extends BackendType {
   publicationInstance: ReportPublicationInstance;
 }
 
-export interface ReportEntityDescription extends BackendType {
-  abstract: string;
-  contributors: Contributor[];
-  date: RegistrationDate;
-  description: string;
-  language: LanguageValues;
-  mainTitle: string;
-  npiSubjectHeading: string;
+export interface ReportEntityDescription extends BaseEntityDescription {
   reference: ReportReference;
```

Comment:
Smak og behag selvfølgelig og vet ikke om våres måte å gjøre det på er noe bedre, men vi har ikke gått for polymorfi, men i stedet satt attributter som ikke alle typer har med ?.
eks. ville vi hatt BaseEntityDescription med reference?: ReportReference.

Reply:
problemet her er at alle typene har ulik `reference`, mens resten er likt. så alternativet er egentlig å ha bare en `EntityDescription` med `reference: JournalReference | BookReference | ReportReference | (...)`. Men får noen problemer med begge løsninger egentlig. mulig det hadde vært like greit å bare ha en type og bruke optionals overalt på sikt ja 🤔